### ISSUE TITLE
Using the calibration.txt file in Python

### ISSUE THREAD

**Issue by iamlegolas** [genekogan/KinectProjectorToolkit#14]
Once the calibration is done, can we use the calibration.txt file in Python? I need to use the Caffe framework to detect objects and project onto their real world coordinates. 

If that's not a possibility, can we call a bunch of processing functions from Pythin scripts? For example I use Python to detect objects using Caffe and then send their coordinates from the Kinect image to the Processing sketch that converts them to real world coordinates and sends them back? 

@genekogan

**Comment by genekogan** [comment]
yes i think in theory, but you'll need to replicate the matrix multiplication.  the equations are from [this blog](http://blog.3dsense.org/programming/kinect-projector-calibration-human-mapping-2/). 

as far as the second idea, processing is just java so you could compile it into a binary for the JVM (like from within eclipse) and i think there are ways of calling it from python. the engineering for all this might be a bit tedious though.

**Comment by iamlegolas** [comment]
Thanks for the prompt response, @genekogan! I've, however, decided to go with a more convenient approach. Since I need both programs on the same computer, I'm going to create a server on the Python side and a client on the processing side to communicate. 

Thought I'd post this here... Might help someone some day...

**Comment by genekogan** [comment]
yes, this is a good approach i think. i was also going to suggest using OSC for the communication.
